egress_proxy/tock.vars.yml

```diff
@@ -8,4 +8,5 @@ proxyallow: |
   uaa.fr.cloud.gov
   google-analytics.com
   api.newrelic.com
+  collector.newrelic.com
```

Comment:
So for my own understanding, `newrelic-admin` calls out to api.newrelic.com but NR proper uses `collector.newrelic.com`? 

Can we double check that these aren't using FedRamp'd endpoints -- e.g., `gov-collector.newrelic.com`?

Reply:
That's a good catch on the FedRamp collector endpoint. I'll confirm whether we're currently configured to send data there.

> So for my own understanding, newrelic-admin calls out to api.newrelic.com but NR proper uses collector.newrelic.com?

That's correct. New Relic's agents send APM data to the collector endpoint. The `newrelic-admin` script's record-deploy command [wraps the New Relic REST API](https://docs.newrelic.com/docs/apm/agents/python-agent/installation/python-agent-admin-script-advanced-usage/#record-deploy) at api.newrelic.com. As far as I can tell, there's not a separate FedRAMP-compliant endpoint for those API calls.